pages/game/roleScreen.tsx

```diff
@@ -8,16 +8,28 @@ interface RoleProps {
 }
 
 const RoleScreen: React.FC<RoleProps> = ({ data }) => {
-  const userId = useUserId();
 
   if (!data) {
     return (<></>);
   }
 
+  const unveilRole = ()=>{
+    const roleCard = document.querySelector(".roleCard")
+    console.log("REVEALED", roleCard, data)
+    if(roleCard && data.url){
+      roleCard.style.backgroundimage = data.url
+    }
+  }
+
   return (
     <>
-      <div className='roleScreen'>
-        {data.ability}
+      <div className="roleCard" >
+        <h1 className="Role">{data.role}</h1>
+        <h3 className='Title'>{data.title}</h3>
```

Comment:
I tend to prefer single quotes instead of double. Its fine either way as long as the file is consistent. classNames should also follow camel case, so Role -> role and Title -> title. I know you don't have any css rules in for it yet, but that could cause a headache as its not readily apparent.